src/main/java/duke/command/DeleteCommand.java

```diff
@@ -0,0 +1,55 @@
+package duke.command;
+
+import duke.ui.Ui;
+import duke.tasklist.TaskList;
+import duke.storage.Storage;
+
+/**
+ * Represents a command used for the deletion of tasks. A <code>DeleteCommand</code> object corresponds to
+ * a command and is represented by the input arguments from the user e.g., <code>"1"</code>
+ */
+public class DeleteCommand extends Commands {
+    public static final String COMMAND_WORDS = "delete";
+    public static final String SUCCESS_MESSAGE = "    Command Executed Successfully";
+    public static final String FAILURE_MESSAGE = "    'Delete' Command Executed Unsuccessfully";
+
+    private static boolean IS_EXIT = false;
+    private String arguments; // In the form of user duke.command
```

Comment:
Similar to the previous commend, perhaps you can put the commends above the statement?